quality_control/models/qc_trigger_line.py

```diff
@@ -14,8 +14,12 @@ class QcTriggerLine(models.AbstractModel):
     user = fields.Many2one(
         comodel_name='res.users', string='Responsible',
         track_visibility='always', default=lambda self: self.env.user)
+    partner = fields.Many2many(
```

Comment:
I don't have any idea why it is partner... cause I was convinced that I put partners... This is crazy